spec/safe_statements_spec.rb

```diff
@@ -100,6 +100,68 @@ def pool_config
         end.not_to raise_error
       end
 
+      it "is configured to run migrations non-transactionally by default" do
+        migration_dir = Dir.mktmpdir
+
+        File.write("#{migration_dir}/0_create_foos.rb", <<~RUBY)
+          class CreateFoos < #{migration_klass}
+            def up
+              safe_create_table :foos
+
+              raise "boom"
+            end
+          end
+        RUBY
+
+        aggregate_failures do
+          expect do
+            migration_klass.suppress_messages do
+              ActiveRecord::MigrationContext.new(
```

Comment:
Can we add comments explaining that our usually way of testing doesn't invoke ActiveRecord's transaction handingling?